src/annotator/anchoring/pdf.js

```diff
@@ -430,7 +428,6 @@ export function anchor(root, selectors) {
  * @param {Object} options -
  *   Options passed to `TextQuoteAnchor` and `TextPositionAnchor`'s
  *   `toSelector` methods.
- * @return {Promise<[TextPositionSelector, TextQuoteSelector]>}
```

Comment:
TS was not happy with this return type, removing it and allowing it to infer seems like the most reasonable solution now that types.js is converted.